packages/automerge-repo-react-hooks/test/useDocument.test.tsx

```diff
@@ -61,15 +71,25 @@ describe("useDocument", () => {
     const { handleA, wrapper } = setup()
     const onDoc = vi.fn()
 
-    render(<Component url={handleA.url} onDoc={onDoc} />, {wrapper})
+    render(<Component url={handleA.url} onDoc={onDoc} />, { wrapper })
     await waitFor(() => expect(onDoc).toHaveBeenLastCalledWith({ foo: "A" }))
   })
 
+  it.only("should immediately return a document if it has already been loaded", async () => {
```

Comment:
Tests are failing because of the `.only` here:
```suggestion
  it("should immediately return a document if it has already been loaded", async () => {
```